Upstream-Status: Backport
Signed-off-by: Ross Burton <ross.burton@intel.com>

From 802b006b3768ddc8c8cdf0af27428e7042c36509 Mon Sep 17 00:00:00 2001
From: nick <oavc-fujie@yandex.com>
Date: Mon, 29 Jun 2015 20:37:26 +0100
Subject: settings: Include pangofc-fontmap.h header on Wayland

In case the X11 backend is not enabled, we still need to include the
pangofc-fontmap.h header file, as we use the Pango/FontConfig API in
both the X11 and Wayland case.

https://bugzilla.gnome.org/show_bug.cgi?id=751625

diff --git a/gtk/gtksettings.c b/gtk/gtksettings.c
index 327faf5..a40cb84 100644
--- a/gtk/gtksettings.c
+++ b/gtk/gtksettings.c
@@ -41,6 +41,7 @@
 
 #ifdef GDK_WINDOWING_WAYLAND
 #include "wayland/gdkwayland.h"
+#include <pango/pangofc-fontmap.h>
 #endif
 
 #ifdef GDK_WINDOWING_BROADWAY
-- 
cgit v0.10.2
